src/bulkExport/getJobStatus.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Handler } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import { BulkExportStateMachineGlobalParameters } from './types';
+import DynamoDbParamBuilder from '../dataServices/dynamoDbParamBuilder';
+import { DynamoDBConverter } from '../dataServices/dynamoDb';
+
+export const getJobStatusHandler: Handler<
+    BulkExportStateMachineGlobalParameters,
+    BulkExportStateMachineGlobalParameters
+> = async event => {
+    const { GLUE_JOB_NAME } = process.env;
+    if (GLUE_JOB_NAME === undefined) {
+        throw new Error('GLUE_JOB_NAME environment variable is not defined');
+    }
+    const glueJobRunId = event.executionParameters?.glueJobRunId;
+    if (glueJobRunId === undefined) {
```

Comment:
Use falsely?

Reply:
Similar as above. I'm ok with just doing an undefined check here. 

This lambda will fail anyways due to the call to Glue if the glueJobRunId is invalid and that's ok. That'd mean that precedent steps in the state machine are generating invalid/incomplete outputs and need to be fixed.

This is just a convenience to detect if outputs from previous steps are missing since that's the most likely dev error when assembling the step functions state machine.